llmfoundry/icl_eval/yamls/mosaic_gpt_eval.yaml

```diff
@@ -26,28 +26,198 @@ load_path: # Add your (optional) Composer checkpoint path here!
 #   sharding_strategy: FULL_SHARD
 #   mixed_precision: PURE
 
+
 icl_tasks:
--
-  label: piqa
-  dataset_uri: # ADD YOUR OWN DATASET URI
-  num_fewshot:
-  - 5
-  batch_size: 16
-  icl_task_type: multiple_choice
-  metric_names:
-  - InContextLearningMultipleChoiceAccuracy
-  prompt_string: '' # this goes at the beginning of each input
-  example_delimiter: '\n' # this goes between fewshot examples
-  continuation_delimiter: ' ' # this separates questions from answers
--
-  label: lambada
-  dataset_uri: # ADD YOUR OWN DATASET URI
-  num_fewshot:
-  - 0
-  batch_size: 16
-  icl_task_type: language_modeling
-  metric_names:
-  - InContextLearningLMAccuracy
-  prompt_string: '' # this goes at the beginning of each input
-  example_delimiter: '\n' # this goes between fewshot examples
-  continuation_delimiter: ' ' # this separates contexts from continuations
+  -
+    label: lambada_openai
+    dataset_uri: local_data/lambada_openai.jsonl 
+    num_fewshot:
+    - 0
+    batch_size: 4
+    icl_task_type: language_modeling
+    metric_names:
+    - InContextLearningLMAccuracy
+    - InContextLearningLMExpectedCalibrationError
+    prompt_string: '' # this goes at the beginning of each input
+    example_delimiter: '\n' # this goes between fewshot examples
+    continuation_delimiter: ' ' # this separates contexts from continuations
+  -
+    label: piqa
+    dataset_uri:  local_data/piqa.jsonl  # ADD YOUR OWN DATASET URI
+    num_fewshot:
+    - 0
+    - 1
+    - 5
+    - 10
```

Comment:
was it a conscious decision to go all the way up to 10?

Reply:
yeah I wanted to max out our perf...Jeopardy keeps increasing up to 10.